### PR TITLE
feat: enable parallelism option for TF apply stage

### PR DESCRIPTION
**Pull request by siiimooon**
Hi,
 
 The parallelism option would come in handy when using providers that do not handle parallell operations gracefully.

### CODE REVIEW

**Comment by chanwit** [comment]
Thank you for the PR @siiimooon 
It is a good addition. I'm taking a look at it and it would be possible to get merged soon.

File: runner/server.go [hunk 1]
```diff
@@ -1005,6 +1005,10 @@ func (r *TerraformRunnerServer) Apply(ctx context.Context, req *ApplyRequest) (*
 		applyOpt = append(applyOpt, tfexec.Target(target))
 	}
 
+	if req.Parallelism != 0 {
```
Comment (chanwit) [inline runner/server.go:1008]:
```suggestion
	if req.Parallelism > 0 {
```

Comment (chanwit) [reply runner/server.go:1008]: It's to prevent the user to specify a negative value.

**Review by chanwit — CHANGES_REQUESTED** [review]

File: api/v1alpha1/terraform_types.go [hunk 2]
```diff
@@ -212,6 +212,11 @@ type TerraformSpec struct {
 	// +optional
 	Targets []string `json:"targets,omitempty"`
 
+	// Parallelism limits the number of concurrent operations of Terraform apply step.
```
Comment (chanwit) [inline api/v1alpha1/terraform_types.go:215]:
```suggestion
	// Parallelism limits the number of concurrent operations of Terraform apply step. Zero (0) means using the default value.
```

**Review by chanwit — CHANGES_REQUESTED** [review]

**Comment by siiimooon** [comment]
Thanks for the feedback, @chanwit. I have implemented both of your suggestions.

**Review by chanwit — APPROVED** [review]
LGTM!
Thank you so much @siiimooon :1st_place_medal:

**Pull request merged**